tests/e2e-tests/specs/front-end/front-end-single-product.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import { activatePlugin } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { createSimpleProduct, createVariableProduct } from "../../utils/components";
+import { CustomerFlow, StoreOwnerFlow } from "../../utils/flows";
+import { uiUnblocked } from '../../utils';
+
+describe( 'Single Product Page', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'woocommerce' );
+		await createSimpleProduct();
+		await createVariableProduct();
+		await StoreOwnerFlow.logout();
```

Comment:
It's probably good practice just to ensure that the rest of the test runs "as a normal customer" vs anything special happening because there is an active admin session/logged in user.

Reply:
IMO, yes, we do need to log out in order to run the rest of the test as a `customer` who is not logged in as an `admin` to match the typical shopping experience on WooCommerce site when someone buys a product.